src/xbps/index.md

```diff
@@ -105,3 +105,13 @@ To get a list of all installed packages, without their version:
 ```
 $ xbps-query -l | awk '{ print $2 }' | xargs -n1 xbps-uhelper getpkgname
 ```
+
+## Verifying RSA keys
+
+When installing Void for the first time, you may get a message from xbps claiming:
+
+```
+<REPO> repository has been RSA signed by <RSA-FINGERPRINT>
+```
```

Comment:
This can also happen if the signing key changes for some reason.

Also, use `xbps-install` there, instead of "xbps".